VAMobile/src/screens/PaymentsScreen/DirectDepositScreen/HowToUpdateDirectDepositScreen/HowToUpdateDirectDepositScreen.tsx

```diff
@@ -25,15 +25,15 @@ function HowToUpdateDirectDepositScreen({ navigation }: HowToUpdateDirectDeposit
         backLabel={t('payments.title')}
         backLabelOnPress={navigation.goBack}
         title={t('directDeposit.title')}>
-        <AlertBox border="warning">
+        <AlertWithHaptics variant="warning">
           <TextView variant="MobileBodyBold" accessibilityRole="header" paragraphSpacing={true}>
             {t('howToUpdateDirectDeposit.alert.title')}
           </TextView>
           <TextView variant="MobileBody" paragraphSpacing={true}>
```

Comment:
Could be partially simplified moving two of these to the `header` and `description` props to AlertWithHaptics.